WordPress/src/main/java/org/wordpress/android/ui/mysite/ImprovedMySiteFragment.kt

```diff
@@ -281,6 +289,19 @@ class ImprovedMySiteFragment : Fragment(R.layout.new_my_site_fragment),
                 CTA_DOMAIN_CREDIT_REDEMPTION
         )
         is AddNewSite -> SitePickerActivity.addSite(activity, action.isSignedInWpCom)
+        is ShowQuickStartDialogOld -> showQuickStartDialog(
+                R.string.quick_start_dialog_need_help_title,
+                R.string.quick_start_dialog_need_help_message,
+                R.string.quick_start_dialog_need_help_button_positive,
+                R.string.quick_start_dialog_need_help_manage_site_button_negative,
+                R.string.quick_start_dialog_need_help_button_neutral
+        )
+        is ShowQuickStartDialogNew -> showQuickStartDialog(
+                R.string.quick_start_dialog_need_help_manage_site_title,
+                R.string.quick_start_dialog_need_help_manage_site_message,
+                R.string.quick_start_dialog_need_help_manage_site_button_positive,
+                R.string.quick_start_dialog_need_help_button_negative
+        )
```

Comment:
Question ❓:  How about these strings are set on the navigation action in the VM itself? Then write a test case that the appropriate dialog is shown with the right set of strings?

Reply:
Such a nice suggestion @ashiagr , on it!

I should start moving everything to view model, including strings, I sometimes miss that and thanks goodness you are here to help me. 🌟  🙏

Reply:
This is now done here: 6b4f8ec1c64bf52b733425db3b2789db7c699400